main/inc/lib/javascript/ckeditor/plugins/toolbarswitch/plugin.js

```diff
@@ -34,6 +34,9 @@ function switchMe(editor, callback) {
 	var origContentCss = editor.config.contentsCss;
```

Comment:
Acabo de probar el plugin y me aparece un error:

Uncaught ReferenceError: CKeditor_OnComplete is not defined

Al parecer no soporta la versión 4.4